radio/src/translations/ua.h

```diff
@@ -0,0 +1,1257 @@
+#define TR_BLUE                         "Блакитний"
+#define TR_GREEN                        "Зелений"
+#define TR_COLOR_PICKER                 "Обирач кольору"
+#define TR_EDIT_THEME_DETAILS           "Редагувати деталі теми"
+#define TR_THEME_COLOR_DEFAULT         "ЗАМОВЧ"
+#define TR_THEME_COLOR_PRIMARY1        "ОСНОВНИЙ1"
+#define TR_THEME_COLOR_PRIMARY2        "ОСНОВНИЙ2"
+#define TR_THEME_COLOR_PRIMARY3        "ОСНОВНИЙ3"
+#define TR_THEME_COLOR_SECONDARY1      "ВТОРИННИЙ1"
+#define TR_THEME_COLOR_SECONDARY2      "ВТОРИННИЙ2"
+#define TR_THEME_COLOR_SECONDARY3      "ВТОРИННИЙ3"
+#define TR_THEME_COLOR_FOCUS           "ФОКУС"
+#define TR_THEME_COLOR_EDIT            "РЕДАГУВАТИ"
+#define TR_THEME_COLOR_ACTIVE          "АКТИВНИЙ"
+#define TR_THEME_COLOR_WARNING         "WARNING"
+#define TR_THEME_COLOR_DISABLED        "ДЕАКТИВОВАНИЙ"
+#define TR_THEME_COLOR_CUSTOM          "КОРИСТУВАЦЬКИЙ"
+#define TR_THEME_CHECKBOX              "Checkbox"
+#define TR_THEME_ACTIVE                "Акстивний"
+#define TR_THEME_REGULAR               "Регулярний"
+#define TR_THEME_WARNING               "Warning"
+#define TR_THEME_DISABLED              "Деактивований"
+#define TR_THEME_EDIT                  "Редагувати"
+#define TR_THEME_FOCUS                 "Фокус"
+#define TR_AUTHOR                       "Автор"
+#define TR_DESCRIPTION                  "Опис"
+#define TR_SAVE                         "Зберігти"
+#define TR_CANCEL                       "Відміна"
+#define TR_EDIT_THEME                   "РЕДАГУВАТИ ТЕМУ"
+#define TR_DETAILS                      "Деталі"
+#define TR_THEME_EDITOR                 "ТЕМИ"
+
+// Main menu
+#define TR_MAIN_MENU_SELECT_MODEL       "Обрати\nМодель"
+#define TR_MAIN_MENU_MANAGE_MODELS      "Керувати\nМоделями"
+#define TR_MAIN_MENU_MODEL_NOTES        "Моделі\nНотатки"
+#define TR_MAIN_MENU_CHANNEL_MONITOR    "Канал\nМонитор"
+#define TR_MAIN_MENU_MODEL_SETTINGS     "Модель\nУстановки"
+#define TR_MAIN_MENU_RADIO_SETTINGS     "Радіо\nУстановки"
+#define TR_MAIN_MENU_SCREEN_SETTINGS    "Екоани\nУстановки"
+#define TR_MAIN_MENU_RESET_TELEMETRY    "Скидання\nТелеметрії"
+#define TR_MAIN_MENU_STATISTICS         "Статистика"
+#define TR_MAIN_MENU_ABOUT_EDGETX       "Про...\nEdgeTX"
+// End Main menu
+
+// Voice in native language
+#define TR_VOICE_ENGLISH                "Англійська"
+#define TR_VOICE_CHINESE                "Китайська"
+#define TR_VOICE_CZECH                  "Чеська"
+#define TR_VOICE_DANISH                 "Данська"
+#define TR_VOICE_DEUTSCH                "Німецька"
+#define TR_VOICE_DUTCH                  "Голандська"
+#define TR_VOICE_ESPANOL                "Іспанська"
+#define TR_VOICE_FRANCAIS               "Французська"
+#define TR_VOICE_HUNGARIAN              "Угорська"
+#define TR_VOICE_ITALIANO               "Італійська"
+#define TR_VOICE_POLISH                 "Польська"
+#define TR_VOICE_PORTUGUES              "Португальська"
+#define TR_VOICE_RUSSIAN                "кцп"
```

Comment:
Change to "Pосійський" or "Російська" (first option would likely be better suitable for a language according to Google).

Reply:
Agree with @rotorman. 
This merge request must not be merged because:

1. There is no "кцп" abbreviation or word in the Ukrainian language. The right variant is only "Російська" according to the Ukrainian language. (I'm native speaker)
2.  The abbreviation "кцп" stands for "кацап". "кацап" is derogatory and offensive name for Russian nationality and Russian speaking people. Please see https://ru.wiktionary.org/wiki/%D0%BA%D0%B0%D1%86%D0%B0%D0%BF
3. This MR violates Github rules https://docs.github.com/en/site-policy/acceptable-use-policies/github-hate-speech-and-discrimination I quote: "GitHub does not tolerate speech that attacks or promotes hate toward an individual or group of people on the basis of who they are, including age, body size, ability, ethnicity, gender identity and expression, level of experience, **nationality**"
4. Russian nationality is the second largest nationality of Ukrainian citizen. So, the author offends many Ukrainians.

Maintainers of this repo, please, be very careful when reviewing MR from this arrogant developer @philmoz in future.
He can write some malicious code.

GitHub is a place where all individuals must feel welcome and safe.

Reply:
FYI
@pfeerick @raphaelcoeffic @3djc

Reply:
> Maintainers of this repo, please, be very careful when reviewing MR from this arrogant developer @philmoz in future. He can write some malicious code.

In future you should do your homework before slinging insults like this around.

I did not create this translation file, and I take no responsibility for its contents.

Reply:
@hacky-swan no one of us understand Ukrainian or Russian, so things happen.
You should be more careful how you talk to others, especially when You obviously do not know the circumstances or the organisation of the project.
@philmoz does outstanding work on the project and the translation was not made by him, but by an ukrainian native speaker.
Due to the situation in Ukraine, I can absolutely understand the frustration and anger most Ukrainians feel.
It still is not right to use deragatory terms in the translation, though. 
At the same time it is not your place to insult other people!

Reply:
@hacky-swan you probably missed the point that @philmoz did not create that translation, and that none of us has any knowledge any of the mentioned languages. So we have to rely on others for that part. It works most of the time, and sometimes it goes south.

Other than that, we all agree that *GitHub is a place where all individuals must feel welcome and safe*. We're with you on that particular point.

Also, we're trying hard to focus on what this project is: an operating system for radio control systems. Nothing more, nothing less.

Reply:
@philmoz excuse me please. I didn't think it was not your translation and you just received it from another person who used offensive words.
@raphaelcoeffic I did not find any references to the author of the original translation in this MR.
So I decided that translator was the MR author.

There were a lot of open source projects in which the malicious code were injected due to political circumstances and passed review.
I hope EdgeTX will be safe from this bad stuff. 
Happy new year!